mrtushardhole-Pune.md

```diff
@@ -0,0 +1,10 @@
+# Orlando, FL
+
+### Restaurants
+
+### Places to visit
+
+- Shanivar wada
+- Beach
```

Comment:
Beach is always on my mind... coundn't think of any other dummy attraction :wink: